src/nvim/api/window.c

```diff
@@ -545,20 +548,29 @@ Dict nvim_win_text_height(Window window, Dict(win_text_height) *opts, Arena *are
     });
   }
 
+  int64_t end_row = INT64_MAX;
```

Comment:
Why is this variable called `end_row`?

Reply:
Well because the row where the loop ends is returned in it, I'll change it though. Seems less confusing the other way around here.